cmd/podmanV2/containers/inspect.go

```diff
@@ -7,6 +7,8 @@ import (
 )
 
 var (
+	containerExistsDescription = `If the named container exists in local storage, podman container exists exits with 0, otherwise the exit code will be 1.`
```

Comment:
Why is this in `inspect.go`?

Reply:
no my code ... someone will get that when inspect is actually done

Reply:
But you are adding it in this PR and it's not used in this file...  Please move it to the file it's used in.

Reply:
done